db/schema.rb

```diff
@@ -53,6 +81,46 @@
     t.datetime "updated_at"
   end
 
+  create_table "blazer_audits", force: :cascade do |t|
```

Comment:
seems that the migration files for these are not in this PR. Rollback and re-run?